OxfordBodleian/Juel-Jensen/BDLaethe31.xml

```diff
@@ -50,35 +611,145 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     <material key="parchment"/>
                                 </support>
                                 <extent>
-                                    <!--<measure unit=""></measure>-->
+                                    <note>Consists of <measure unit="strips">3</measure> strips sewn together.</note>
+                                    <measure type="weight" unit="g">100</measure>
                                     <dimensions type="outer" unit="mm">
                                         <height>1900</height>
                                         <width>200</width>
                                     </dimensions>
-                                    <measure type="weight" unit="g">100</measure>
+                                    <dimensions type="outer" unit="mm" xml:id="strip1">
+                                        <height>705</height>
+                                        <width>195</width>
+                                    </dimensions>
+                                    <dimensions type="outer" unit="mm" xml:id="strip2">
+                                        <height>650</height>
+                                        <width>200</width>
+                                    </dimensions>
+                                    <dimensions type="outer" unit="mm" xml:id="strip3">
+                                        <height>578</height>
+                                        <width>200</width>
+                                    </dimensions>
                                 </extent>
+                                <condition key="good">The upmost part of the scroll is damaged, affecting <ref target="#d1"/>. The bottom part
```

Comment:
`<ref target="#d1">Miniature 1</ref>` (same for the later ref in this sentence), I think